src/helpers/strip-private.py

```diff
@@ -119,6 +119,7 @@ def strip_lines(rules: tuple) -> None:
         (True, re.compile(r'(shared-secret-key-file|ca-cert-file|cert-file|dh-file|key-file|client) (\S+)'), r'\1 xxxxxx'),
         # Strip IPSEC secrets
         (True, re.compile(r'pre-shared-secret \S+'), 'pre-shared-secret xxxxxx'),
+        (True, re.compile(r'\ssecret \S+'), ' secret xxxxxx'),
```

Comment:
Why is a leading whitespace required and not only `secret`?

Reply:
Generic word which might later appear at the end of another string like `use-secret` so figured I'd capture whole

Reply:
Good intention but this will happen:

`use-secret foo` CLI node will become ` secret xxxxxx` so the output of `strip-private` invalidates the configuration.

Please just make it as simple as possible and use:

`(True, re.compile(r'secret \S+'), 'secret xxxxxx'),`